modules/UI/videolayout/LargeVideoManager.js

```diff
@@ -242,6 +245,18 @@ export default class LargeVideoManager {
                 const tracks = state['features/base/tracks'];
                 const videoTrack = getVideoTrackByParticipant(tracks, participant);
 
+                // Remove track streaming status listener from the old track and add it to the new track,
+                // in order to stop updating track streaming status for the old track and start it for the new track. 
+                if (this.jitsiTrack) {
+                    this.jitsiTrack.off(JitsiTrackEvents.TRACK_STREAMING_STATUS_CHANGED, this.handleTrackStreamingStatusChanged);
+                    APP.store.dispatch(trackStreamingStatusChanged(this.jitsiTrack, this.jitsiTrack.getTrackStreamingStatus?.()));
+                }
+                if (videoTrack && !videoTrack.local) {
+                    this.jitsiTrack = videoTrack?.jitsiTrack;
```

Comment:
I don't think we need the here `?`, we already checked if `videoTrack` is defined above

Reply:
fixed